packages/manager/apps/ips/src/pages/byoip/Byoip.context.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+export const ByoipContext = React.createContext<{
+  ipRir: string;
+  setIpRir: React.Dispatch<React.SetStateAction<string>>;
+  selectedRegion: string;
+  setSelectedRegion: React.Dispatch<React.SetStateAction<string>>;
+  ipRange: string;
+  setIpRange: React.Dispatch<React.SetStateAction<string>>;
+  as: string;
+  setAs: React.Dispatch<React.SetStateAction<string>>;
+  selectedToken: string;
+  setSelectedToken: React.Dispatch<React.SetStateAction<string>>;
+}>({
+  ipRir: '',
+  setIpRir: () => {},
+  selectedRegion: '',
+  setSelectedRegion: () => {},
+  ipRange: '',
+  setIpRange: () => {},
+  as: '',
+  setAs: () => {},
+  selectedToken: '',
+  setSelectedToken: () => {},
+});
+
+export const ByoipContextProvider: React.FC<React.PropsWithChildren> = ({
+  children,
+}) => {
+  const [ipRir, setIpRir] = React.useState<string>('');
+  const [selectedRegion, setSelectedRegion] = React.useState<string>('');
+  const [selectedToken, setSelectedToken] = React.useState<string>('');
+  const [ipRange, setIpRange] = React.useState<string>('');
+  const [as, setAs] = React.useState<string>('');
+
+  const value = React.useMemo(
+    () => ({
+      ipRir,
+      setIpRir,
```

Comment:
if you go back and select another ipRir, you should delete the selectedRegion etc.